### ISSUE TITLE
LOWBB pairs setting now correct in GUI

### ISSUE THREAD

**Issue by spoon2eric** [mehtadone/PTFeeder#275]
PT Version: 1.2.6.20
PTF Version: Latest

When setting LOWBB to for example -1 (or any value for that matter) in pairs file for any pair, GUI shows -10. Restarts, json deletes, deleting all trading files and allowing PTF to push new files, etc do not fix the issue.

![ada-pairs](https://user-images.githubusercontent.com/35353533/37679550-3930ec26-2c4f-11e8-80c1-3ae5627650bb.jpg)

![appsettings](https://user-images.githubusercontent.com/35353533/37679692-b568bb16-2c4f-11e8-87c3-f200bd745010.jpg)

**Comment by spoon2eric** [comment]
during "boring" mode, the BuyValue should be "-1".. it's always "-10" for some reason in the PT GUI.

**Comment by mehtadone** [comment]
Are you sure you are looking at the same file? Looking at the comments above the pair specific config, the price trends etc are all different too

**Comment by spoon2eric** [comment]
The issue can be closed. 

However, somehow I was running a different PT jar when starting up PT. The start up command I “was” using was just -jar and the PT jar file. Even though I was in the correct PT folder, PT was not reading the pairs file. I don’t know if somehow PT was executing out of my backup folder or what, but I was getting zero errors. When I specifically gave the FULL path to the -jar command it works perfectly. Like I said, I was in the correct upgraded PT folder when I ran the start command but did not provide a full path for the jar, just simply -jar ProfitTrailer.jar. How it knew which jar to run is something you might be able to explain. Once I provided the full path, everything started to work. 

Either way. Ticket can be closed. 

Thank you and looking forward to PT 2.0!!

Eric W
